### PR TITLE
Add badges to README.md

### PR DESCRIPTION
**Pull request by kojix2**
Hello @kanga333 

I added a gem badge and a docs badge.
Some people don't like the badges, so please close freely if you don't want them.
I'm not web developer and I don't know Fluent well.
So this is the only contribution I can make.

Good luck.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # fluent-plugin-s3-arrow
 
+[![Gem Version](https://badge.fury.io/rb/fluent-plugin-s3-arrow.svg)](https://badge.fury.io/rb/fluent-plugin-s3-arrow)
+[![Docs](https://img.shields.io/badge/docs-latest-blue.svg)](https://rubydoc.info/gems/fluent-plugin-s3-arrow)
```
Comment (kanga333) [inline README.md]:
```suggestion
```

This plugin is not a library, so it's better not to link to rdoc.

**Review by kanga333 — COMMENTED** [review]
Thank you! I wanted to add a badge to the repository to make it look better.

File: README.md (hunk 1 above)
Comment (kojix2) [reply README.md]: Thanks done!

**Comment by kanga333** [comment]
Thank you! I'll merge.

**Pull request merged**